project/plugins.sbt

```diff
@@ -5,6 +5,8 @@ addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "2.3.1")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.3")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.14.2")
+addSbtPlugin("org.typelevel" % "sbt-typelevel-mergify" % "0.7.7")
```

Comment:
This is a nitpick, but if you need to modify this file anyway to remove `sbt-scala-native`, could you please move this line up so it's immediately under the `sbt-typelevel` reference above? In other words, make this line the new line 3. That will keep the `sbt-typelevel*` dependencies together, which hopefully will be easier to read and help reduce future merge conflicts.